### PR TITLE
feat: add shortcut to crop from question game

### PR DESCRIPTION
**Pull request by teolemon**
### What
- feat: add shortcut to crop from question game

### CODE REVIEW

File: src/off.ts [hunk 1]
```diff
@@ -82,7 +82,11 @@ const offService = {
       lang === "en" ? "" : "-" + lang
     }.openfoodfacts.org/cgi/product.pl?type=edit&code=${barcode}`;
   },
-
+  // TODO: Create a UI button allowing to jump to the Crop page by barcode from the question game
+  getLogoCropsByBarcodeUrl(barcode) {
+  return `https://hunger.openfoodfacts.org/logos/search?barcode==${barcode}`;
```
Comment (teolemon) [inline src/off.ts:87]:
```suggestion
  return `https://hunger.openfoodfacts.org/logos/search?barcode=${barcode}`;
```

File: src/off.ts [hunk 2]
```diff
@@ -82,7 +82,11 @@ const offService = {
       lang === "en" ? "" : "-" + lang
     }.openfoodfacts.org/cgi/product.pl?type=edit&code=${barcode}`;
   },
-
+  // TODO: Create a UI button allowing to jump to the Crop page by barcode from the question game
```
Comment (teolemon) [inline src/off.ts:85]:
```suggestion
```

**Comment by alexfauquette** [comment]
If I get it correctly you would like to got on `https://hunger.openfoodfacts.org/logos/product-search?code=3017620422003` and get all the logos from `3017620422003` to annotate?

**Comment by teolemon** [comment]
@alexfauquette 
Yep. We should probably also add a deep link to infinity annotation for the current tag.

File: src/pages/insights/InsightsGrid.jsx [hunk 3]
```diff
@@ -150,7 +156,18 @@ const InsightGrid = ({ filterState = {}, setFilterState }) => {
...
                 </Tooltip>
               }
+            />,
+                        <GridActionsCellItem
+              component="a"
+              href={getLogoCropsByBarcodeUrl(params.row.barcode)}
+              label={t("insights.view_product")}
+              icon={
+                <Tooltip title={t("insights.view_product")}>
+                  <VisibilityIcon />
```
Comment (teolemon) [inline src/pages/insights/InsightsGrid.jsx:166]:
```suggestion
              label={t("insights.view_crops_for_this_product")}
              icon={
                <Tooltip title={t("insights.view_crops_for_this_product")}>
                  <VisibilityIcon />
```

File: src/pages/questions/ProductInformation.jsx [hunk 4]
```diff
@@ -110,6 +110,17 @@ const ProductInformation = (props) => {
...
+            <Button
+        size="small"
+        component={Link}
+        target="_blank"
+        href={offService.getLogoCropsByBarcodeUrl(question.barcode)}
+        variant="contained"
+        startIcon={<EditIcon />}
+        sx={{ ml: 2, minWidth: 150 }}
+      >
+        {t("questions.edit")}
```
Comment (teolemon) [inline src/pages/questions/ProductInformation.jsx:123]:
```suggestion
        {t("insights.view_crops_for_this_product")}
```

**Review by alexfauquette — APPROVED** [review]

**Pull request merged**